docs/Plugins.rst

```diff
@@ -182,6 +182,8 @@ Options:
 
 :detail:      Print full name, date of birth, date of curse and some status
               info (some vampires might use fake identities in-game, though).
+:cprobe:      Print some information from creature probe plugin, displaying
```

Comment:
I think we should name this `id` or `ids`, because that's more reflective what is being printed (and is somewhat more consistent with the other existing options here). `cprobe` is a tool that some output has been copied from, but people using `cursecheck` may not be familiar with it.

Other than that, this looks like it should work to me!